### ISSUE TITLE
Can we setup a cloudflare worker to generate resources?

### ISSUE THREAD

**Issue by danblah** [ResetNetwork/recoding#4]
something like recoding.reset.tech/resrouce-export that, behind cloudflare access, that pulls from zotero, creating a csv/josn for importing into notion?

**Comment by danblah** [comment]
Another thought is to use a cron trigger to keep the file up to date: https://developers.cloudflare.com/workers/platform/cron-triggers

**Comment by rudietuesdays** [comment]
Yup, can use the script we're working on in the `citations-feed` branch
